mpf/devices/multiball.py

```diff
@@ -291,6 +291,54 @@ def add_a_ball(self):
             self.balls_added_live += 1
             self.machine.game.balls_in_play += 1
             self.source_playfield.add_ball(balls=1)
+            self.add_a_ball_timer_start()
+
+    def add_a_ball_timer_start(self) -> None:
+        """Start the timer for add a ball ball save.
+
+        This is started when multiball add a ball is triggered if configured,
+        and the default timer is not still running.
+        """
+        if self.shoot_again:
+            # if main ball save timer is running, don't run this timer
+            return
+        self.shoot_again = True
+
+        shoot_again_ms = self.config['add_a_ball_shoot_again'].evaluate([])
+        if not shoot_again_ms:
+            # No shoot again. Just stop multiball right away
+            self.stop()
```

Comment:
I guess you also want to add `return` here. Otherwise, we will still post the events below.

Reply:
Agreed.  I merged the two methods for normal multiball into one, since there is less to handle, but screwed that up.  So I have added the return statement, and then removed the else, as all things past will run if they don't return.